blog/content/post/2019-02-14-enabling-cpb-on-pcengines-apu2.md

```diff
@@ -431,4 +431,4 @@ looking for someone who can boot your product by leveraging advanced features of
 used hardware platform, feel free to
 [boot a call with us](https://calendly.com/3mdeb/consulting-remote-meeting) or
 drop us email to `contact<at>3mdeb<dot>com`. Are You interested in similar
-content? Feel free to [sign up for our newsletter](https://newsletter.3mdeb.com/subscription/PW6XnCeK6)
+content? Feel free to [sign up for our newsletter](https://3mdeb.com/subscribe/3mdeb_newsletter.html)
```

Comment:
This link give 404 at this point

Reply:
It's still in draft, I suppose it's not yet ready on prod